src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -229,7 +229,7 @@ void releaseMongoConnection(DBClientBase* connection, std::auto_ptr<DBClientCurs
 #else
 #if MONGO_DRIVER_LEGACY_1_0_6_OR_NEWER
```

Comment:
Not really... that (1.0.6) was the driver version that introduces kill(). However, at the end we are not going to use kill() (details in #1568).

Anyway, this part of the code needs a little refactoring. Coming in a next commit.